code/modules/events/blob.dm

```diff
@@ -14,11 +14,10 @@
 	var/mob/C
 	if(candidates.len)
 		C = pick(candidates)
-		Blob = new /obj/effect/blob/core(T, 200)
+		Blob = new /obj/effect/blob/core(T, 200, null, 2, 0, 1)

```

Comment:
The third argument to `new` here is where you're supposed to provide the client to put directly in control of the blob. You can get rid of all your changes to `core.dm`, get rid of the `create_overmind` call below this, and use `(T, 200, C.client)` to put the chosen candidate in direct control.